CI broken with "conflicting implementation in crate `time`"

CI is currently failing on every PR with [E0119](https://doc.rust-lang.org/error_codes/E0119.html).

The `time` crate published [0.3.48](https://crates.io/crates/time/0.3.48) which adds a trait impl that collides, via [coherence](https://rustc-dev-guide.rust-lang.org/coherence.html) overlap checker, with fea-rs's blanket impl at

https://github.com/googlefonts/fontc/blob/d201554c1cd63eef4f1f3c3bffbbf88b57010d03/fea-rs/src/common.rs#L44-L48

Upstream bug https://github.com/time-rs/time/issues/783

Since we don't commit Cargo.lock, CI resolves the latest.

The error only occurs when passing --all-features to cargo:

```
    error[E0119]: conflicting implementations of trait
      `From<...HourBase>` for type `<...HourBase as ...ModifierValue>::Type`
      --> fea-rs/src/common.rs:44:1
       = note: conflicting implementation in crate `time`:
               - impl From<HourBase> for <HourBase as ModifierValue>::Type;
       = note: upstream crates may add a new impl of trait
               `Into<fontdrasil::types::GlyphName>` for type `HourBase` in
               future versions
```

We could pin time to =0.3..47 or maybe narrow the blanket `impl<T: Into<GlyphName>> From<T> for GlyphIdent`...

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,6 +21,10 @@ rayon = "1.6"
 icu_properties = "=2.1"
 smallvec = {version = "1.15", features = ["const_new"]}
 
+# TEMPORARY: see https://github.com/googlefonts/fontc/issues/2042
+# and https://github.com/time-rs/time/issues/783.
+time = "=0.3.47"
+
 # fontations etc
 write-fonts = { version = "0.49.1", features = ["serde", "read"] }
 skrifa = { version = "0.43.2", features = ["traversal"] }

--- a/fontc/Cargo.toml
+++ b/fontc/Cargo.toml
@@ -49,6 +49,10 @@ rayon = { workspace = true, optional = true }
 # just for fontc!
 crossbeam-channel = "0.5.6"
 
+# Not used directly: this only exists so the workspace pin on `time` (see the
+# root Cargo.toml) takes effect on resolution. Remove together with that pin.
+time = { workspace = true }
+
 clap = { workspace = true, optional = true }
 
 [dev-dependencies]

